ChurchCRM/CRM
# Feature/calendar subscribe api

#### What's this PR do?
Allows events with the "publicly visible" flag to be accessed as an iCal format for easy subscription in Google Calendar, Apple Calendar, or Outlook.

#### Screenshots (if appropriate)

#### What Issues does it Close?

Closes

#### What are the relevant tickets?

#### Any background context you want to provide?
ChurchCRM is our Church's central database for events (public and private), and I want congregants to be able to easily see public events on their mobile devices and existing personal calendars 

#### How should this be manually tested?
1) ensure that ```bEnableExternalCalendarAPI``` is enabled
2) Create a few events with the "publicly visible" box checked
3) Follow [Google's Instructions](https://support.google.com/calendar/answer/37100?co=GENIE.Platform%3DDesktop&hl=en) for adding a iCal by URL.  supply the url https://CRM.YourChurhc.org/external/calendar/ics
